app/views/proposal_mailer/proposal_published_notification.html.erb

```diff
@@ -9,7 +9,8 @@
   </p>
   <% unless @request_type == 'delete' %>
     <div class="center">
-      <%= link_to 'View Collection', collection_url(@concept_id, revision_id: @revision_id), class: 'btn btn-blue' %>
+      <% # not sure if there needs to be some revision parameter in the URL %>
```

Comment:
not sure if there needs to be some revision parameter in the URL, my understanding is that the most recent revision is the one that would be returned